### ISSUE TITLE
Issue during Database migration

### ISSUE THREAD

**Issue by disk91** [helium/console#776]
1. Are you using Helium hosted Console or running your own open source Console?
Self Hosted

2. If you are on Helium hosted Console, which url are you currently on? 
console.helium-iot.eu

3. If you are experiencing bugs in Helium hosted Console, please specify your Organization name and Device ID (ex. 3483050b-284d-47e2-8eda-e087fc03a753).

Please describe your issue:

When upgrading the console, the database migration scripts is failling. As a consequence, the next migrations are dropped and the database is not correctly ugraded.

```
helium_console | 14:16:24.172 [info] alter table labels
helium_console | 14:16:24.175 [info] alter table devices
helium_console | 14:16:24.180 [info] == Migrated 20210428210734 in 0.0s
helium_console | 14:16:24.189 [info] == Running 20210505174257 Console.Repo.Migrations.AddTypeColToAlertEvents.change/0 forward
helium_console | 14:16:24.189 [info] alter table alert_events
helium_console | 14:16:24.191 [info] == Migrated 20210505174257 in 0.0s
helium_console | 14:16:24.199 [info] == Running 20210506220027 Console.Repo.Migrations.AddAppEuiToOrg.up/0 forward
helium_console | 14:16:24.199 [info] alter table organizations
helium_console | ** (Postgrex.Error) ERROR 42701 (duplicate_column) column "default_app_eui" of relation "organizations" already exists
helium_console |     (ecto_sql) lib/ecto/adapters/sql.ex:593: Ecto.Adapters.SQL.raise_sql_call_error/1
helium_console |     (elixir) lib/enum.ex:1336: Enum."-map/2-lists^map/1-0-"/2
helium_console |     (ecto_sql) lib/ecto/adapters/sql.ex:686: Ecto.Adapters.SQL.execute_ddl/4
helium_console |     (ecto_sql) lib/ecto/migration/runner.ex:343: Ecto.Migration.Runner.log_and_execute_ddl/3
helium_console |     (ecto_sql) lib/ecto/migration/runner.ex:117: anonymous fn/6 in Ecto.Migration.Runner.flush/0
helium_console |     (elixir) lib/enum.ex:1948: Enum."-reduce/3-lists^foldl/2-0-"/3
helium_console |     (ecto_sql) lib/ecto/migration/runner.ex:116: Ecto.Migration.Runner.flush/0
helium_console |     _build/prod/rel/console/lib/console-0.0.1/priv/repo/migrations/20210506220027_add_app_eui_to_org.exs:12: Console.Repo.Migrations.AddAppEuiToOrg.up/0
```
when removing concerned script, jump to the next one
```
helium_console | 14:26:56.388 [info] create check constraint dev_eui_must_be_16_chars on table devices
helium_postgres | 2021-08-18 14:26:56.390 UTC [35] ERROR:  constraint "dev_eui_must_be_16_chars" for relation "devices" already exists
helium_postgres | 2021-08-18 14:26:56.390 UTC [35] STATEMENT:  ALTER TABLE "devices" ADD CONSTRAINT "dev_eui_must_be_16_chars" CHECK (LENGTH(dev_eui) = 16)
helium_console | ** (Postgrex.Error) ERROR 42710 (duplicate_object) constraint "dev_eui_must_be_16_chars" for relation "devices" already exists
```

basically scripts should support re-execution.

**Comment by vicmgs** [comment]
Get into the docker container:
docker exec -it helium_console /bin/bash

Re-run migration in bash:
_build/prod/rel/console/bin/console eval "Console.Release.migrate"

**Comment by disk91** [comment]
Thank you for the commande, it will simplify my problem
BUT, This was not my point, If the scripts can be played multiple times when the console is starting, they should not crash the migration process and continue.

**Comment by disk91** [comment]
Other question : 
How to execute one of the specific migration script ?
I always get 
```
[info] Already up
```
when running the given command

**Comment by vicmgs** [comment]
Already up means all your migrations have already run once. Migration scripts are not meant to be run multiple times when console is starting, that's just how ecto works. If the migration stops on a script when console is starting, it means something in your db doesn't conform to the rules and you should fix it before running the later scripts. 

I have never had to rerun migrations individually, maybe this can help:
https://hexdocs.pm/ecto_sql/Ecto.Migrator.html
